deps/GraphBLAS/Source/GxB_Matrix_tuple_iter.c

```diff
@@ -348,7 +349,10 @@ GrB_Info GxB_MatrixTupleIter_next
 		*col = A->i[nnz_idx] ;
 	}
 	if(val) {
-		GrB_Index offset = nnz_idx * iter->size;
+		bool iso;
+		GxB_Matrix_iso(&iso, A); // return iso status of a matrix
```

Comment:
Please add `bool iso` to the `GxB_MatrixTupleIter` struct, thus saving a call to `GxB_Matrix_iso` on each iteration.
what we want is:
```c
GrB_Index offset = (iter->iso) ? 0 : nnz_idx * iter->size;
```